shell-ui/package.json

```diff
@@ -47,7 +47,7 @@
     "webpack-dev-server": "^3.11.2"
   },
   "dependencies": {
-    "@scality/core-ui": "github:scality/core-ui.git#v0.22.2.1",
+    "@scality/core-ui": "github:scality/core-ui.git#695c63c",
```

Comment:
do we really want to keep the commit hash rather than deploying a new version ?

Reply:
Yes. You are right. We should tag a new version for the core-ui. 
I am just waiting a bit to see if I can get a review from @scality/doc about the wording in the 401 unauthorized page.